logging/README.md

```diff
@@ -0,0 +1,54 @@
+# ELK Logging for Developer Environments
+
+This allows us to use centralized logging with advanced search and UI features to debug
+issues seen within developer environments.
+
+## Start Logging Sidecars
```

Comment:
could you add more info about setting ELK logging up as well as explanations about all the scripts and usage. [load-test-tools](https://github.com/AudiusProject/load-test-tools) is a good example

Reply:
The information that exists in the README is currently all the information needed for someone to:

* Launch the Filebeat/Metricbeat sidecars.
* Navigate to Kibana.
* View pre-configured Kibana dashboards.
* Filter the logs based on fields and indexes.
* Get started using Elastic's getting started guides.

Additionally, if someone were to build another image, the additional scripts in the README (`./bin/build-image.sh` and `./bin/dev-image.sh` (which is simply a wrapper around `./bin/build-image.sh`)) are the only one/two other scripts that will ever need to be called directly during image-development. These scripts should have sufficient comments that explain when and why the other scripts are called.

In order to provide a better developer experience, I had chosen to create an easy-to-understand README for ELK consumers and have all infrastructure-related details be consolidated within a single script `build-image.sh` which is referenced within the README.

However, I did add two new links to the official Elasticsearch documentation for Filebeat and Metricbeat specifically.

Was there any specific information that you were looking for but could not find?

Reply:
@joaquincasares i actually think your README is really good for a developer who's purely a consumer of this feature. the piece i wanted to see a bit more is for someone who wants to extend this functionality like someone who works on this infra in the future. they'll have to understand what this does and right now it's really hard. for example for someone coming into the bin/ folder with no context it's very hard to understand what's going on

Reply:
@dmanjunath good catch!

I've added eb38a1993d035c53ae6a04932e765d5fd572dd4b which should address your concerns. Do please let me know if there's any additional information you'd like to see.